src/rpc/blockchain.cpp

```diff
@@ -1165,6 +1165,11 @@ UniValue getblockchaininfo(const JSONRPCRequest& request)
             + HelpExampleRpc("getblockchaininfo", "")
         );
 
+    if (!chainActive.Tip())
```

Comment:
This can be an assert too; RPC isn't available before the `InitBlockIndex` call, after which chainActive cannot be empty.